docker/sane-doc-reports/Dockerfile

```diff
@@ -1,17 +1,16 @@
 
-FROM demisto/python3:3.8.3.8715
+FROM demisto/python3-deb:3.8.3.8715
+WORKDIR "/app"
 
-COPY requirements.txt .
+SHELL ["/bin/bash", "-c"]
+
+RUN update-ca-certificates
+RUN apt update -yq && apt upgrade -y && apt autoremove -yq \
+    && apt install -y nodejs npm
 
-RUN apk add --no-cache --virtual .build-deps gcc musl-dev \
- && pip install cython \
- && apk del .build-deps
-RUN apk add --update --no-cache g++ gcc libxslt-dev freetype-dev jpeg-dev zlib-dev tzdata
-RUN apk --update add --no-cache --virtual .build-dependencies build-base wget git \
-  && pip install --no-cache-dir -r requirements.txt \
-  && apk del .build-dependencies
-RUN ln -fs /usr/share/zoneinfo/UTC /etc/localtime
+RUN npm install svgexport -g --unsafe-perm=true
```

Comment:
You should install a specific version using `svgexport@<version>` or use a package-lock.json file (I see you have one already).